src/views/PasswordReset/PasswordReset.css

```diff
@@ -0,0 +1,86 @@
+:root {
+  --background-color-placeholder: #F9F9F9;
+  --background-color: white;
+  --text-color-black: #0A0A0B;
+  --text-color-grey: #B9B7BD;
+  --button-color: #FB7C5C;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-image: linear-gradient(120deg, var(--background-color) 0%, var(--button-color) 100%);
+  font-family: "Lato", sans-serif;
+  min-height: 100vh;
+}
+
+.password-reset-form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  max-width: 500px;
+  min-width: 300px;
+  max-height: 700px;
+  width: 80%;
+  height: 60%;
+  margin: 100px auto;
+}
+
+.password-reset {
+  border-radius: 0 10px 10px 0;
+  position: relative;
+}
+
+.password-reset-form h1 {
+  font-size: 6vw;
```

Comment:
The header gets too big on big screens. 
What I like doing most is defining the font size at `rem` unit (remember to define the base font-size at root) and then only changing the root font size at 1 or 2 media queries.

You can choose any other solution you like, just make sure that the font sizes are looking nice at different resolutions (also the big ones) :)